CHANGELOG.md

```diff
@@ -1,5 +1,106 @@
+- Search stats: do not double-count ([#2708](https://github.com/MultiQC/MultiQC/pull/2708))
+- Escape values passed to HTML properties (e.g. `val` in tables) ([#2706](https://github.com/MultiQC/MultiQC/pull/2706))
+- Fix re-loading explicit user configs in interactive sessions ([#2704](https://github.com/MultiQC/MultiQC/pull/2704))
+- Fix saving table plots to file ([#2735](https://github.com/MultiQC/MultiQC/pull/2735))
+- Fix `--pdf` option to generate `multiqc_report.pdf` ([#2733](https://github.com/MultiQC/MultiQC/pull/2733))
+- Fix file search performance regression ([#2762](https://github.com/MultiQC/MultiQC/pull/2762))
+- Fix handling module href string ([#2739](https://github.com/MultiQC/MultiQC/pull/2739))
+- Custom content:
+  - Cast names to str instead of asserting ([#2769](https://github.com/MultiQC/MultiQC/pull/2769))
+  - Fix parsing custom content tables with numerical samples ([#2705](https://github.com/MultiQC/MultiQC/pull/2705))
+  - Section order and custom content order: allow skip the `-module` suffix from ID ([#2770](https://github.com/MultiQC/MultiQC/pull/2770))
+
+### Updates
+
+- Search patterns: allow multiple values for `contents` ([#2696](https://github.com/MultiQC/MultiQC/pull/2696))
+- Custom content:
+  - Allow dict input to heatmap ([#2761](https://github.com/MultiQC/MultiQC/pull/2761))
+  - Allow multiple custom content to general stats table ([#2727](https://github.com/MultiQC/MultiQC/pull/2727))
+- Plots:
+  - Defer render of plots if number of samples > `config.plots_defer_loading_numseries` ([#2759](https://github.com/MultiQC/MultiQC/pull/2759), [#2777](https://github.com/MultiQC/MultiQC/pull/2777), [#2773](https://github.com/MultiQC/MultiQC/pull/2773), [#2774](https://github.com/MultiQC/MultiQC/pull/2774))
+  - Line plot: show markers when num of data points < `config.lineplot_number_of_points_to_hide_markers` (=50) ([#2760](https://github.com/MultiQC/MultiQC/pull/2760))
+  - Line plot: smooth by default to 500 ([#2776](https://github.com/MultiQC/MultiQC/pull/2776))
+  - Scale exported plot fonts with `config.plots_export_font_scale` ([#2758](https://github.com/MultiQC/MultiQC/pull/2758))
+  - Improve performance loading large tables in browser ([#2737](https://github.com/MultiQC/MultiQC/pull/2737))
+  - Fix toolbox highlight in line plots ([#2724](https://github.com/MultiQC/MultiQC/pull/2724))
+- Interactive: use module anchor or lowercase module name to key data ([#2741](https://github.com/MultiQC/MultiQC/pull/2741))
+- More helpful validation error: print parent class name if applicable ([#2709](https://github.com/MultiQC/MultiQC/pull/2709))
+
+### New modules
+
+- VG stats ([#2690](https://github.com/MultiQC/MultiQC/pull/2690))
+- ngs-bits ([#2231](https://github.com/MultiQC/MultiQC/pull/2231))
+- Pairtools ([#1148](https://github.com/MultiQC/MultiQC/pull/1148))
+- nanoq ([#2723](https://github.com/MultiQC/MultiQC/pull/2723))
+- ganon ([#1935](https://github.com/MultiQC/MultiQC/pull/1935))
```

Comment:
Can we add the one-line descriptions for each tool please? And maybe link the name to the tool homepage..